CONTRIBUTING.md

```diff
@@ -24,19 +24,23 @@ so.
 
 ## Development
 Note before contributing that everything done here is under the [MIT](https://opensource.org/license/mit/) license.
+
 ### Naming
 Naming is an important matter, and it shouldn't be changed unless necessary.
 
 Game **names** should be added as they appear on steam (or other storefront
-if not listed there) with the release year appended in brackets (except when the
-release year is already part of the name).  
+if not listed there) with also the release year included as `games.release_year`. 
 If there is a mod that needs to be added (or it adds the support for server
 queries for the game), its name should be composed of the game name, a separating
 **bracket**, the mod name and the release year as specified previously
-(e.g. `Grand Theft Auto V - FiveM (2013)`).
+(e.g. `Grand Theft Auto V - FiveM`).
+
+Older IDs exist because of a naming system update. Having them as an option, 
+guarantees that other softwares will still function after a library update.
```

Comment:
This isn't really related to what this document is, the line could be added in the migrating document tho.